testdata/SampleClientImpl.txt

```diff
@@ -18,12 +19,11 @@ import com.fasterxml.jackson.databind.ObjectMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
 import java.net.URI;
 import java.util.HashSet;
```

Comment:
so looks like importing HasSet is optional and this test case tested when  it is true, do we need another test when it is false?